tasks/nginx.yml

```diff
@@ -58,3 +58,9 @@
     dest: "/etc/nginx/sites-enabled/wifi_login_pages"
     state: link
   notify: restart nginx
+
+- name: Configure nginx log rotation
+  template:
+    src: logrotate.d/openwisp-nginx.j2
+    dest: "/etc/logrotate.d/openwisp-nginx"
```

Comment:
Let's call it `owlp-nginx`? Otherwise, it will overwrite the file created by ansible-openwisp2 

https://github.com/openwisp/ansible-openwisp2/blob/8ef0e8ea7f52bedd6286b48ae9e1bcb9e57183f5/tasks/nginx.yml#L82-L86